Lib/test/test_asyncio/test_sslproto.py

```diff
@@ -429,6 +429,7 @@ def test_start_tls_server_1(self):
 
         server_context = test_utils.simple_server_sslcontext()
         client_context = test_utils.simple_client_sslcontext()
+        client_context.options |= ssl.OP_NO_TLSv1_3
```

Comment:
Can we add a comment explaining this line?

Reply:
Done in c9a8f58

Reply:
Can we please restrict this to FreeBSD?

@1st1 In the long run, we should have parametrized tests to test TLS 1.2 and TLS 1.3 behavior. The protocols behave differently on mulitple accounts. For client cert auth, we even need to have TLS 1.3 with and without PHA.

Reply:
Done in f777fa5f2bd16ac8d60416eaa64eb9d2cf84ffac